tests/test_forge_data.py

```diff
@@ -0,0 +1,593 @@
+        "4636f6d70616374647261776572730a312e31382d342e312e3000116465636f7261746976655f626c6f636b7305322e312e30000a626f74616e79"
+        "706f747306382e302e3132000b6674626261636b7570733206312e302e31370209636f66685f636f726508312e362e342e32310767656e6572616"
+        "c01310102086d636a74796c69620b312e31382d362e302e3135086d636a74796c696203312e3001000869737061776e657203312e300009657665"
+        "7279636f6d700c312e31382e322d312e352e37000a6a6569747765616b657207332e302e302e38000974657272616c69746807302e304e4f4e450"
+        "20d6d696e696e676761646765747306312e31312e30146d61696e5f6e6574776f726b5f6368616e6e656c013200020c6372616674747765616b65"
+        "7207392e312e313937046d61696e05312e302e3000020b616b6173686963746f6d6506312e352d3230046d61696e0131000405666f72676503414"
+        "e590c746965725f736f7274696e6703312e30000573706c697403312e3101020e636f6c6f7373616c63686573747305312e382e330c6368616e6e"
+        "656c5f6d61696e05312e302e3001020673656c656e650d312e31382e322d312e31372e39076e6574776f726b01310000136472697070796c6f616"
+        "4696e6773637265656e05312e362e34030e6372616674696e67747765616b73076e6574776f726b03312e300004096d696e65637261667406312e"
+        "31382e320a756e726567697374657204464d4c330108726567697374657204464d4c3301000c7465727261626c656e64657210312e31382e322d3"
+        "12e312e302e3130320018736f70686973746963617465646261636b7061636b7376680f312e31382e322d312e302e342e3132000b6d6f75736574"
+        "7765616b7303414e590208746974616e69756d05332e352e36076e6574776f726b03312e300101046a616465000d637265617465747765616b657"
+        "208322e302e302e3137020e656173795f76696c6c61676572730d312e31382e322d312e302e31300764656661756c7405312e302e300102057069"
+        "70657a0c312e31382e322d312e312e350764656661756c7405312e302e300100076963656265726703414e590108666c79776865656c02066d616"
+        "e746c6506312e392e3237076e6574776f726b013100000965636f6c6f6769637305312e372e330205717561726b07332e322d333538046d61696e"
+        "013100020c786165726f6d696e696d61700732322e31312e31046d61696e03312e3001000670696770656e05382e302e3102096661737462656e6"
+        "36805362e302e32076368616e6e656c05342e362e30010209706f6c796d6f7270680b312e31382e322d302e3434046d61696e013100000a617574"
+        "6f7265676c696206312e372d3533020e73746f72616765647261776572730631302e322e310c6d61696e5f6368616e6e656c013100000c666c757"
+        "86e6574776f726b7307372e302e372e38000a6e656f6e63726166743203322e320208656e657263656c6c07302e304e4f4e45076e6574776f726b"
+        "05302e302e300002096170706c65736b696e0c322e342e302b6d63312e31380473796e63013101010b66657272697465636f7265020e6d6f64756"
+        "c6172726f757465727308392e312e312d39330c6d61696e5f6368616e6e656c0132000014726566696e656473746f726167656164646f6e730530"
+        "2e382e32000a6f70656e6c6f616465720631322e302e3102097468655f7661756c7411312e31382e322d322e302e31302e383639076e6574776f7"
+        "26b06302e32362e300001156d6f6465726e75693a666c75786e6574776f726b730337303700"
+    )
+
+
+def test_decode_forge_data() -> None:
```

Comment:
It's enough to test it in `test_java.py`.

Reply:
Removed in 0eb8031